### ISSUE TITLE
Could not find module `ember-assign-polyfill`

### ISSUE THREAD

**Issue by oskarrough** [shipshapecode/ember-flatpickr#83]
Hi, I'm getting the following error with ember 2.10.0 and ember-flatpickr 1.1.0 when I visit a route that has ember-flatpickr in it.

> Uncaught Error: Could not find module `ember-assign-polyfill` imported from `ember-flatpickr/components/ember-flatpickr`

As soon as I run `yarn add ember-assign-polyfill` it works. But I suppose that shouldn't be necessary. Also since I'm above Ember 2.4. Any ideas?

**Comment by lypborges** [comment]
Same error here. I need to change the package.json on the ember-flatpickr.

```javascript
 "dependencies": {
    "ember-cli-babel": "^5.1.7",
    "ember-cli-node-assets": "0.1.6",
    "ember-assign-polyfill": "1.0.0",
    "flatpickr": "^2.2.3"
  },
```

And add `    "ember-assign-polyfill": "1.0.0",`

**Comment by rwwagner90** [comment]
Sorry guys, I had `ember-assign-polyfill` in devDependencies, but I think it needed to be in dependencies. I moved it and released version 1.1.1 with the change. Please give it a shot and let me know if this is resolved.

**Comment by oskarrough** [comment]
No worries. Version 1.1.2 works here. Thank you for the quick response!

Just for the record, when do you need to use devDependencies and when dependencies for ember-cli addons?
